File: hackerrank.com/Algorithms/Python/PlusMinus.py
def plusMinus(arr):
    # Write your code here
    countList = [0, 0, 0]
    for a in arr:
        if a > 0:
            countList[0] += 1
        elif a < 0:
            countList[1] += 1
        else:
            countList[2] += 1
    
    for i in countList:
        print(i / len(arr))

File: hackerrank.com/Algorithms/C/PlusMinus.c
void plusMinus(int arr_count, int* arr) {
    float countPositive=0, countNegative=0, countZeros=0;
    
    for (int i=0; i < arr_count; i++)
    {
        if (arr[i] > 0)
            countPositive++;
        else if (arr[i] < 0)
            countNegative++;
        else
            countZeros++;    
    }
    
    printf("%f\n%f\n%f", countPositive / arr_count, countNegative / arr_count, countZeros / arr_count);  
}